src/business_logic/transaction/objects/internal_invoke_function.rs

```diff
@@ -340,27 +337,35 @@ mod tests {
         };
 
         // Instantiate CachedState
-        let state_reader = InMemoryStateReader::new(HashMap::new(), HashMap::new());
-        let mut state = CachedState::new(state_reader, None);
-
-        // Initialize state.contract_classes
-        state.set_contract_classes(HashMap::new()).unwrap();
-
+        let mut state_reader = InMemoryStateReader::default();
         // Set contract_class
         let class_hash: [u8; 32] = [1; 32];
         let contract_class =
             ContractClass::try_from(PathBuf::from("starknet_programs/fibonacci.json")).unwrap();
+        // Set contact_state
```

Comment:
Here too, 
can we insert the same values used in the previous version of the test

Reply:
Done